_posts/2021-08-29-china_computationa_psychiatry_hack.markdown

```diff
@@ -0,0 +1,18 @@
+---
+layout: post
+title: "1st Computational Psychiatry Hack @ China 2021"
+date: 2021-08-29 09:00:00
+big: 1
+project_categories: Events 2021
+image: china_hack_2021.jpg
+description: Get hands dirty, learn by doing!
+---
+
+URL: [1st Computational Psychiatry Hack](https://www.heywhale.com/home/competition/612310e47db76d0017825f3b/?from=cp)
+
+The 1st Computational Psychiatry Hack @ China is jointly organized by the Chinese Computational Psychiatry Network (CCPN), the Sophon Academy and Heywhale. In this program, the participants will join as a team to learn computational modeling. They will reproduce the analysis of a published paper in the computational psychiatry (CP) [1], under the guidance of experienced experts. All the participants will have a precious experience to know the CP field by doing a completed project with teammates and chance to win impressive scholarships! 
```

Comment:
Is the reference or link to [1] missing somewhere?

Reply:
Oh sure! Thank you so much for pointing it out. Sure it is a reference to a paper. Will make the correction. Thank you.

Reply:
Added! Thank you.